compose-files/docker-compose.yml

```diff
@@ -22,12 +22,49 @@ services:
   ###
   mongodb:
     image: mongo:${MONGO_DOCKER_TAG:-latest} ## change tag in version.env
-    container_name: mongodb
+    container_name: mongodb-primary
     ports:
       - "27017:27017"
     volumes:
-      - data-mongodb:/data/db
+      - data-mongodb-primary:/data/db
+    depends_on:
+      - mongodb-secondary
+      - mongodb-arbiter
```

Comment:
The mongodb-secondary and mongodb-arbiter shoud be depends on the mongodb-primary, i think.
The secondarys at mongodb need the primary, the primary dosn't need the secondarys.

Reply:
discussion solfed, is required for a proper start of mongodb